rocket-app/rocket-list/src/main/java/com/example/rocket_list/presentation/RocketViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.rocket_list.presentation
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.rocket_repo.model.Rocket
+
+class RocketViewModel : ViewModel() {
+    private val _rockets = MutableLiveData<List<Rocket>>()
+    val rockets: LiveData<List<Rocket>>
+        get() = _rockets
+
+    fun getRockets() {
+        _rockets.postValue(com.example.rocket_repo.data.rockets)
+    }
+}
```

Comment:
pridat state + mapping, ktery se pouzije na urovni Compose (tzn i v Flow tady ve view modelu)

```suggestion
    data class State(
        val rockets: List<RocketItemState> = emptyList()
    ) {
        data class RocketItemState(
            val id: String,
            val name: String,
            val firstFlight: String,
        )
    }
}
```